### PR TITLE
XRWH-170 Styled download button pop-up according to the mock-up

### PR DESCRIPTION
**Pull request by yuliia-biletska**

### CODE REVIEW

File: plugins/rocks.xml.webhelp/xsl/topic.xsl [hunk 1]
```diff
@@ -239,13 +239,12 @@
...
-                                            <a href="{$output-pdf-full-path}" target="_blank">
-                                                <div class="download-output">
-                                                    <xsl:call-template name="getVariable">
-                                                        <xsl:with-param name="id" select="'Download common PDF'"/>
-                                                    </xsl:call-template>
-                                                </div>
-                                            </a>
+                                            <div class="download-output">
+                                                <a href="{$output-pdf-full-path}" target="_blank"/>
+                                                <xsl:call-template name="getVariable">
```
Comment (ibarinova) [inline plugins/rocks.xml.webhelp/xsl/topic.xsl]: As far as I see, this text should be inside &lt;a&gt; tag. Is this button working now?

Comment (yuliia-biletska) [reply plugins/rocks.xml.webhelp/xsl/topic.xsl]: i fixed this

**Review by ibarinova — APPROVED** [review]
Approved

**Review by vasyl-krupa — APPROVED** [review]
Looks good

**Pull request merged**